product_docs/docs/pgd/5/manual/05-creating-cluster.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Step 5 - Creating the PGD Cluster
+navTitle: Creating the Cluster
+deepToC: true
+---
+
+## Creating the PGD cluster
+
+* **Create connection strings for each node**.  
+For each node we want to create a connection string which will allow PGD to perform replication.
+
+ The connection string is a key/value string which starts with a `host=` and the IP address of the host (or if you have resolvable named hosts, the name of the host).
+
+ That is followed by the name of the database; `dbname=bdrdb` as we created a `bdrdb` database when [installing the software](04-installing-software).
+
+ We recommend you also add the port number of the server to your connection string as `port=5444` for EDB Postgres Advanced Server and `port=5432` for EDB Postgres Extended and Community PostgreSQL.
+
```

Comment:
We could give the example connection string being used in TPA.

Reply:
My understanding is that TPA flattens the EPAS port to be 5432 anyway, so this would be inappropriate for a manual install of EPAS with PGD.